src/index.js

```diff
@@ -229,7 +256,9 @@ class List {
         );
 
         // mark active
-        itemEl.classList.toggle(this.CSS.settingsButtonActive);
+        if (item.name === "unordered" || item.name === "ordered") {
```

Comment:
create an array outside the class `listTypes = ["ordered", "unordered"]`;
and then use it like `listTypes.contains(item.name)`